### PR TITLE
2.x

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

File: src/bda/plone/ajax/setuphandlers.py [hunk 1]
```diff
@@ -19,4 +19,4 @@ class HiddenProducts(object):
     def getNonInstallableProducts(self):
         """Do not show on QuickInstaller's list of installable products.
         """
-        return ['bda.plone.ajax:install-base']
+        return []
```
Comment (rnixx) [inline src/bda/plone/ajax/setuphandlers.py]: setuphandlers.py is obsolete with this PR. no QI uninstallable profiles left

File: setup.py [hunk 2]
```diff
@@ -18,9 +18,8 @@
     classifiers=[
         "Environment :: Web Environment",
         "Framework :: Plone",
-        "Framework :: Plone :: 4.3",
-        "Framework :: Plone :: 5.0",
         "Framework :: Plone :: 5.1",
+        "Framework :: Plone :: 5.2",
         "Framework :: Zope2",
         "License :: OSI Approved :: GNU General Public License (GPL)",
         "Operating System :: OS Independent",
```
Comment (jensens) [inline setup.py]: Classifiers for Zope 4 and Python versions are missing.

File: src/bda/plone/ajax/setuphandlers.py (hunk 1 above)
Comment (jensens) [reply src/bda/plone/ajax/setuphandlers.py]: well, we need a HiddenProfile for uninstall, don't we?

Comment (rnixx) [reply src/bda/plone/ajax/setuphandlers.py]: sure, once there is an uninstall profile...

Comment (agitator) [reply src/bda/plone/ajax/setuphandlers.py]: fixed

File: setup.py (hunk 2 above)
Comment (agitator) [reply setup.py]: fixed

File: setup.py [hunk 3]
```diff
@@ -21,6 +21,11 @@
         "Framework :: Plone :: 5.1",
         "Framework :: Plone :: 5.2",
         "Framework :: Zope2",
+        "Framework :: Zope4",
```
Comment (jensens) [inline setup.py]: this classifier does not exist, see https://pypi.org/pypi?:action=list_classifiers

Comment (agitator) [reply setup.py]: fixed

File: src/bda/plone/ajax/setuphandlers.py [hunk 4]
```diff
@@ -10,13 +9,4 @@ class HiddenProfiles(object):
...
-
-
-@implementer(QuickInstaller.INonInstallable)
-class HiddenProducts(object):
-
-    def getNonInstallableProducts(self):
-        """Do not show on QuickInstaller's list of installable products.
-        """
-        return ['bda.plone.ajax:install-base']
+        return ['bda.plone.ajax:default']
```
Comment (jensens) [inline src/bda/plone/ajax/setuphandlers.py]: `default`? Did you moreover mean `uninstall` here?

**Pull request merged**